gemfiles/Gemfile.rubocop-next

```diff
@@ -4,5 +4,4 @@ eval_gemfile "../Gemfile"
 
 gem "rubocop", ">= 0.87"
 
-# latest rubocop-shopify release (1.0.4) is not compatible with rubocop 0.89, but this unreleased version is
-gem "rubocop-shopify", git: "https://github.com/Shopify/ruby-style-guide.git", ref: "1da32fee92427b3b04c0d1ede3f762889137a9e5"
+gem "rubocop-shopify", ">= 1.0.5"
```

Comment:
```suggestion
gem "rubocop-shopify", ">= 1.0.6"
```

Reply:
```suggestion
gem "rubocop-shopify", ">= 1.0.7"
```

@volmer just released 1.0.7. https://github.com/Shopify/ruby-style-guide/releases/tag/v1.0.7 It requires rubocop ~1.4.

Reply:
🤩 🤩

Reply:
Updated.